src/current/v24.3/data-resilience.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Data Resilience
+summary: An overview of how CockroachDB deployments can achieve resilience with built-in features and tooling.
+---
+
+CockroachDB provides built-in [_high availability_](#high-availability) features and [_disaster recovery_](#disaster-recovery) tooling to achieve operational resiliency in various deployment topologies and use cases. To maximize uptime, high availability features ensure continuous access to data without interruption even in the presence of failures or disruptions. Disaster recovery tools allow for recovery from major incidents to minimize downtime and data loss.
```

Comment:
maybe silly but i'd maybe add (HA) and (DR) after the first mentions of these terms so search picks it up, also we say HA/DR a lot

Reply:
topologies could link to https://deploy-preview-18997--cockroachdb-docs.netlify.app/docs/v24.3/topology-patterns

Reply:
these could be bullets for the lazy readers among us like me:

 To maximize uptime,
 
 - High availability features ensure continuous access to data without interruption even in the presence of failures or disruptions
 - Disaster recovery tools allow for recovery from major incidents to minimize downtime and data loss.

Reply:
'failures or disruptions' could link to https://www.cockroachlabs.com/docs/stable/demo-fault-tolerance-and-recovery

Reply:
Thanks, added the links, changed to bullets, and also added the acronyms. I also changed in a couple of places on the page to use the acronyms where I felt it made sense (open to adjusting these as I've now looked at it for too long and can't decide).